### ISSUE TITLE
Read time no working with cyrillic 

### ISSUE THREAD

**Issue by Axeny** [daveis/manifesto-anchor-theme#2]
Small typo in the title, it should be Read time noT working with cyrillic 

If the post is written in different language other than English(for example I used Bulgarian) Read Time doesn't work as intended. 

You can see the issue here - http://gizmodjo.com/blog/posts/hey-world

**Comment by daveis** [comment]
@Axeny Did you ever get to a solution? I have not had time to investigate, and I am not sure how to support cyrillic with my custom function.

**Comment by Axeny** [comment]
Nope, I couldn't :( It somehow doesn't work with cyrillic. I will also have
to say that I use markdown as main editor in anchor, so maybe this ruins
the things, too?

2014/1/14 daveis notifications@github.com

> Did you ever get to a solution? I have not had time to investigate, and I
> am not sure how to support cyrillic with my custom function.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/daveis/manifesto-anchor-theme/issues/2#issuecomment-32307171
> .

## 

Ïîçäðàâè
Áàõàð Ìóñòàôà
<a href="http://o-writing.com">Áëîãúò ìè çà ïèñàíåòî</a>

**Comment by daveis** [comment]
I believe I need to use a multibyte string function to support the Cyrillic characters? But that's just a guess.

Also, since you are using Markdown you probably need to change line 113 in functions.php

```
$mycontent = article_html();
```

Should be changed to

```
$mycontent = article_markdown();
```
